test/e2e/playwright.config.ts

```diff
@@ -26,7 +26,9 @@ export default defineConfig( {
 	/* Retry on CI only */
 	retries: process.env.CI ? 1 : 0,
 	/* Workers should use what is available */
-	workers: 1,
+	workers: process.env.CI ? 1 : '100%',
```

Comment:
@adimoldovan is your change correct that TeamCity will only use a single worker, eg. run all tests in a particular project sequentially? (no parallelism)

Reply:
Yes, I forgot to switch that back. Let's add 50% for CI, to keep is in sync with the [current Jest config](https://github.com/Automattic/wp-calypso/blob/fae92cb8c2d87f66d3082ca2884bcfa256567696/.teamcity/settings.kts#L64).

```suggestion
	workers: process.env.CI ? '50%' : '100%',
```

Reply:
Thanks, I updated this in my new branch in #105586